modules/configuring-egress-proxy.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/configure-egress-proxy-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-egress-proxy_{context}"]
+= Configuring the cluster wide egress proxy on the AWS Load Balancer Operator
+
+After configuring the cluster wide egress proxy on the AWS Load Balancer Operator in the {product-title} cluster, Operator Lifecycle Manager (OLM) automatically updates all the deployments of the Operators with the environment variables such as `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY`.
+
+You must trust a custom Certificate Authority (CA) and create the OLM subscription of the AWS Load Balancer Operator.
+
+. Create the config map containing the CA bundle in the `aws-load-balancer-operator` namespace by running the following command:
```

Comment:
```suggestion
. Create the config map which will contain the CA bundle in the `aws-load-balancer-operator` namespace by running the following command:
```

Reply:
Addressed - made it into the present tense